pkg/model/manifest.go

```diff
@@ -263,6 +263,51 @@ func NewDeployInfo() *DeployInfo {
 	}
 }
 
+//GetManifestV1 gets a manifest from a path or search for the files to generate it
+func GetManifestV1(manifestPath string) (*Manifest, error) {
+	cwd, err := os.Getwd()
```

Comment:
Same logic as GetManifestV2. Can you encapsulate this into a function called `loadManifestFromPathFile` or something similar?